pipelines/DECam/ApVerify.yaml

```diff
@@ -11,6 +11,7 @@ imports:
   - location: $AP_PIPE_DIR/pipelines/DECam/ApPipe.yaml
     include:
       - prompt
+      - afterburner
```

Comment:
You'd need to also exclude from `_ingredients/ApVerify.yaml`, else you have double definitions.